components/cam/src/physics/rrtmgp/radiation.F90

```diff
@@ -108,6 +108,12 @@ module radiation
    ! Disabled when value is less than 0.
    real(r8) :: fixed_total_solar_irradiance = -1
 
+   ! The RRTMGP warnings are printed when the state variables need to be limted,
```

Comment:
In your current implementation, you are only controlling warning messages for *temperature*, but we have limiters for optical properties as well. I'd suggest either renaming the namelist variable to `rrtmgp_enable_temperature_warnings`, or making this flag control the optical property limiters as well. I think I'd prefer the former, and then if we later want to disable warnings for optical properties too we can add a flag to do that.

Reply:
will do.